ci/cloudbuild/dockerfiles/fedora-35-cxx20.Dockerfile

```diff
@@ -0,0 +1,207 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM fedora:35
```

Comment:
Should `fedora-34-cxx20.Dockerfile` go away?

Reply:
Yes, but I think it should be in a future PR. If I remove it now I break anything in flight.

Reply:
Ack.